tests/system.py

```diff
@@ -1066,7 +1067,7 @@ def test_create_signed_read_url_v4_w_access_token(self):
         client = iam_credentials_v1.IAMCredentialsClient()
         service_account_email = Config.CLIENT._credentials.service_account_email
         name = client.service_account_path("-", service_account_email)
-        scope = ["https://www.googleapis.com/auth/devstorage.read_write"]
+        scope = ["https://www.googleapis.com/auth/cloud-platform"]
```

Comment:
This seems like a lot of access. @jkwlui @frankyn is this scope needed for this?

Reply:
(we don't have to block merging on this as it is a test, but if this is needed it seems like a large scope for narrow use?)

Reply:
FWIW, I can confirm that both tests fail on master with a 403 without this patch.

Reply:
Could you narrow the scope down to: `https://www.googleapis.com/auth/iam`

Documented at the bottom of the following document: https://cloud.google.com/iam/docs/reference/credentials/rest/v1/projects.serviceAccounts/signBlob

Reply:
To clarify, you can do join both:

```
scope = ['https://www.googleapis.com/auth/devstorage.read_write', 'https://www.googleapis.com/auth/iam']
```